tests/test-cases/Group6-VIC-Machine/6-14-Update.robot

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 6-14 - Verify vic-machine update function
+Resource  ../../resources/Util.robot
+Test Teardown  Run Keyword If Test Failed  Cleanup VIC Appliance On Test Server
```

Comment:
You don't need to qualify this with 'if test failed' and you can eliminate the cleanup at the end of the script.